platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/problems/ConfigurationCacheProblemsSummary.kt

```diff
@@ -214,14 +238,18 @@ class Summary(
      * This should exclude problems with severity [ProblemSeverity.SuppressedSilently], which are still
      * accounted for in [totalProblemCount] and shown in the HTML report, but intentionally omitted
      * from the console to keep output noise low during graceful degradation.
+     *
+     * Also, these problems are console-unique (but not necessarily report unique, which take the full property trace chain into account).
      */
     private
     val consoleProblemCauses: Map<ProblemCause, ProblemSeverity>,
 
     /**
      * Count of problems that should appear in the HTML report.
      */
-    val uniqueProblemCount: Int,
+    @get:VisibleForTesting
```

Comment:
❓ 
You can access internal field in tests, why do we need `@VisibleForTesting`?

Reply:
For documentation ("why isn't this thing private?")